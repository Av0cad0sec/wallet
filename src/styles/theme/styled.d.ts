import 'styled-components'
import { ThemeType } from 'grommet'

/* This is the suggested way of declaring theme types */
declare module 'styled-components' {
  export interface DefaultTheme extends ThemeType {}
}
