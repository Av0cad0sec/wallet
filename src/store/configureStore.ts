import { routerMiddleware } from 'connected-react-router'
import { configureStore, getDefaultMiddleware } from '@reduxjs/toolkit'
import { createInjectorsEnhancer } from 'redux-injectors'
import createSagaMiddleware from 'redux-saga'

import { createReducer, history } from './reducers'
import { RootState } from 'types'
import { fatalErrorActions } from 'app/state/fatalerror'

export function configureAppStore(state?: Partial<RootState>) {
  const sagaMiddleware = createSagaMiddleware({
    onError: (error, info) => {
      store.dispatch(
        fatalErrorActions.setError({
          message: error.message,
          stack: error.stack,
          sagaStack: info.sagaStack,
        }),
      )
    },
  })
  const { run: runSaga } = sagaMiddleware

  // Create the store with saga middleware
  const middlewares = [sagaMiddleware, routerMiddleware(history)]

  const enhancers = [
    createInjectorsEnhancer({
      createReducer,
      runSaga,
    }),
  ]

  const store = configureStore({
    reducer: createReducer(),
    middleware: [...getDefaultMiddleware(), ...middlewares],
    devTools:
      /* istanbul ignore next line */
      process.env.NODE_ENV !== 'production' || process.env.PUBLIC_URL.length > 0,
    enhancers,
    preloadedState: state,
  })

  return store
}
