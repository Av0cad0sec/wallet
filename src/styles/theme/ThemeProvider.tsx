import { Grommet, ThemeType } from 'grommet'
import * as React from 'react'

import { useThemeSlice } from './slice'

const defaultGrommetTheme: ThemeType = {
  global: {
    font: {
      family: 'Rubik',
    },
  },
}
export const ThemeProvider = (props: { children: React.ReactChild }) => {
  useThemeSlice()

  // const theme = useSelector(selectTheme);
  return (
    // <OriginalThemeProvider theme={theme}>
    <Grommet full theme={defaultGrommetTheme}>
      {React.Children.only(props.children)}
    </Grommet>
    // </OriginalThemeProvider>
  )
}
