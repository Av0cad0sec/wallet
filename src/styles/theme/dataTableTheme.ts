import { IDataTableStyles, ITheme } from 'react-data-table-component'

export const dataTableTheme = {
  background: { default: 'false' },
  highlightOnHover: {
    default: '#88888833',
    text: 'false',
  },
  text: {
    primary: 'false',
    secondary: 'false',
    disabled: 'false',
  },
  sortFocus: {
    default: 'false',
  },
  divider: { default: '#AAAAAAaa' },
} as Partial<ITheme>

export const dataTableCustomStyles: IDataTableStyles = {
  headCells: {
    style: {
      fontSize: '18px',
    },
  },
  rows: {
    style: {
      fontSize: '16x',
    },
  },
}
