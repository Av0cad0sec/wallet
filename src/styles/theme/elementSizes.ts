export const sidebarSmallSizeLogo = 30
export const sidebarMediumSizeLogo = 40
export const mobileToolbarZIndex = 3
export const mobileToolbarButtonsZIndex = mobileToolbarZIndex + 1
export const buildPreviewBannerZIndex = mobileToolbarZIndex - 1

export const mobileFooterNavigationHeight = '4rem'
