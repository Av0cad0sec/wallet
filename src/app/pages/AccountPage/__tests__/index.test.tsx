import { render } from '@testing-library/react'
import * as React from 'react'
import { Provider } from 'react-redux'
import { ThemeProvider } from 'styles/theme/ThemeProvider'
import { createMemoryHistory } from 'history'
import { configureAppStore } from 'store/configureStore'

import { AccountPage } from '..'
import { ConnectedRouter } from 'connected-react-router'

const renderPage = (store, history) =>
  render(
    <Provider store={store}>
      <ThemeProvider>
        <ConnectedRouter history={history}>
          <AccountPage />
        </ConnectedRouter>
      </ThemeProvider>
    </Provider>,
  )

describe('<AccountPage  />', () => {
  let store: ReturnType<typeof configureAppStore>

  beforeEach(() => {
    store = configureAppStore()
  })

  it.skip('should match snapshot', () => {
    const history = createMemoryHistory()
    history.push('/account/account_address')
    const page = renderPage(store, history)
    expect(page.container.firstChild).toMatchSnapshot()
  })
})
