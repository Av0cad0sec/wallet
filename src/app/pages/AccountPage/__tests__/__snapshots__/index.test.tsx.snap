// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AccountPage  /> should match snapshot 1`] = `
.c9 {
  display: inline-block;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  width: 18px;
  height: 18px;
  fill: currentColor;
  stroke: currentColor;
}

.c9 g {
  fill: inherit;
  stroke: inherit;
}

.c9 *:not([stroke])[fill='none'] {
  stroke-width: 0;
}

.c9 *[stroke*='#'],
.c9 *[STROKE*='#'] {
  stroke: inherit;
  fill: none;
}

.c9 *[fill-rule],
.c9 *[FILL-RULE],
.c9 *[fill*='#'],
.c9 *[FILL*='#'] {
  fill: inherit;
  stroke: none;
}

.c0 {
  font-family: Rubik,sans-serif;
  font-size: 18px;
  line-height: 24px;
  background-color: #ffffff;
  color: #16213E;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 12px;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  margin-bottom: 12px;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  background-color: #FFFFFF;
  color: #16213E;
  border: solid 1px #EDEDED;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  border-radius: 5px;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex: 1 1;
  -ms-flex: 1 1;
  flex: 1 1;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 12px;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 1 1;
  -ms-flex: 1 1;
  flex: 1 1;
}

.c15 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c17 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.c18 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  max-width: 100%;
  -webkit-flex: 1 0;
  -ms-flex: 1 0;
  flex: 1 0;
  -webkit-flex-basis: 32ex;
  -ms-flex-preferred-size: 32ex;
  flex-basis: 32ex;
}

.c19 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
  height: 100%;
  padding: 24px;
}

.c21 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  border-bottom: solid 1px rgba(0,0,0,0.33);
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c27 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  margin-top: 24px;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c29 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  max-width: 100%;
  -webkit-flex: 3 0;
  -ms-flex: 3 0;
  flex: 3 0;
  -webkit-flex-basis: 80ex;
  -ms-flex-preferred-size: 80ex;
  flex-basis: 80ex;
}

.c30 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  margin: 0px;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c31 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  background-color: #FFFFFF;
  color: #16213E;
  border: solid 1px #EDEDED;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 48px;
  border-radius: 5px;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  max-width: 100%;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding-right: 12px;
  border-radius: 5px;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-bottom: solid 1px #EDEDED;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  width: 690px;
  -webkit-flex: 1 1;
  -ms-flex: 1 1;
  flex: 1 1;
  padding-bottom: 12px;
}

.c24 {
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  -webkit-align-self: stretch;
  -ms-flex-item-align: stretch;
  align-self: stretch;
  height: 24px;
}

.c11 {
  font-size: inherit;
  line-height: normal;
  font-weight: bold;
  word-break: break-word;
}

.c12 {
  font-size: inherit;
  line-height: normal;
}

.c16 {
  font-size: 12px;
  line-height: 18px;
  color: #0500e2;
  font-weight: 600;
}

.c20 {
  margin-left: 12px;
  margin-right: 12px;
  margin-top: 6px;
  margin-bottom: 6px;
  font-size: 18px;
  line-height: 24px;
}

.c14 {
  font-size: 18px;
  line-height: 24px;
}

.c32 {
  font-size: 26px;
  line-height: 32px;
  max-width: 624px;
  font-weight: 600;
  overflow-wrap: break-word;
}

.c8 {
  display: inline-block;
  box-sizing: border-box;
  cursor: pointer;
  font: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  margin: 0;
  background: transparent;
  overflow: visible;
  text-transform: none;
  color: inherit;
  outline: none;
  border: none;
  padding: 0;
  text-align: inherit;
  line-height: 0;
  padding: 12px;
}

.c8:focus {
  outline: none;
  box-shadow: 0 0 2px 2px #00A9FF;
}

.c8:focus > circle,
.c8:focus > ellipse,
.c8:focus > line,
.c8:focus > path,
.c8:focus > polygon,
.c8:focus > polyline,
.c8:focus > rect {
  outline: none;
  box-shadow: 0 0 2px 2px #00A9FF;
}

.c8:focus::-moz-focus-inner {
  border: 0;
}

.c8:focus:not(:focus-visible) {
  outline: none;
  box-shadow: none;
}

.c8:focus:not(:focus-visible) > circle,
.c8:focus:not(:focus-visible) > ellipse,
.c8:focus:not(:focus-visible) > line,
.c8:focus:not(:focus-visible) > path,
.c8:focus:not(:focus-visible) > polygon,
.c8:focus:not(:focus-visible) > polyline,
.c8:focus:not(:focus-visible) > rect {
  outline: none;
  box-shadow: none;
}

.c8:focus:not(:focus-visible)::-moz-focus-inner {
  border: 0;
}

.c28 {
  display: inline-block;
  box-sizing: border-box;
  cursor: pointer;
  font: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  margin: 0;
  background: transparent;
  overflow: visible;
  text-transform: none;
  border: 2px solid #0500e2;
  border-radius: 4px;
  color: #16213E;
  padding: 4px 22px;
  font-size: 18px;
  line-height: 24px;
  background-color: #0500e2;
  color: #F8F8F8;
  border-radius: 4px;
  -webkit-transition-property: color,background-color,border-color,box-shadow;
  transition-property: color,background-color,border-color,box-shadow;
  -webkit-transition-duration: 0.1s;
  transition-duration: 0.1s;
  -webkit-transition-timing-function: ease-in-out;
  transition-timing-function: ease-in-out;
  font-weight: bold;
}

.c28:hover {
  box-shadow: 0px 0px 0px 2px #0500e2;
}

.c28:focus {
  outline: none;
  box-shadow: 0 0 2px 2px #00A9FF;
}

.c28:focus > circle,
.c28:focus > ellipse,
.c28:focus > line,
.c28:focus > path,
.c28:focus > polygon,
.c28:focus > polyline,
.c28:focus > rect {
  outline: none;
  box-shadow: 0 0 2px 2px #00A9FF;
}

.c28:focus::-moz-focus-inner {
  border: 0;
}

.c28:focus:not(:focus-visible) {
  outline: none;
  box-shadow: none;
}

.c28:focus:not(:focus-visible) > circle,
.c28:focus:not(:focus-visible) > ellipse,
.c28:focus:not(:focus-visible) > line,
.c28:focus:not(:focus-visible) > path,
.c28:focus:not(:focus-visible) > polygon,
.c28:focus:not(:focus-visible) > polyline,
.c28:focus:not(:focus-visible) > rect {
  outline: none;
  box-shadow: none;
}

.c28:focus:not(:focus-visible)::-moz-focus-inner {
  border: 0;
}

.c23 {
  box-sizing: border-box;
  font-size: inherit;
  font-family: inherit;
  border: none;
  -webkit-appearance: none;
  background: transparent;
  color: inherit;
  width: 100%;
  padding-left: 5px;
  padding-right: 5px;
  padding-top: 12px;
  padding-bottom: 12px;
  font-weight: 600;
  margin: 0;
  border: 1px solid rgba(0,0,0,0.33);
  border-radius: 4px;
  outline: none;
  border: none;
}

.c23::-webkit-input-placeholder {
  color: #AAAAAA;
}

.c23::-moz-placeholder {
  color: #AAAAAA;
}

.c23:-ms-input-placeholder {
  color: #AAAAAA;
}

.c23::-webkit-search-decoration {
  -webkit-appearance: none;
}

.c23::-moz-focus-inner {
  border: none;
  outline: none;
}

.c23:-moz-placeholder,
.c23::-moz-placeholder {
  opacity: 1;
}

.c23::-webkit-input-placeholder {
  font-size: 14px;
}

.c23::-moz-placeholder {
  font-size: 14px;
}

.c23:-ms-input-placeholder {
  font-size: 14px;
}

.c23::placeholder {
  font-size: 14px;
}

.c26 {
  box-sizing: border-box;
  font-size: inherit;
  font-family: inherit;
  border: none;
  -webkit-appearance: none;
  background: transparent;
  color: inherit;
  width: 100%;
  padding-left: 5px;
  padding-right: 5px;
  padding-top: 12px;
  padding-bottom: 12px;
  font-weight: 600;
  margin: 0;
  border: 1px solid rgba(0,0,0,0.33);
  border-radius: 4px;
  outline: none;
  border: none;
  padding-right: 48px;
}

.c26::-webkit-input-placeholder {
  color: #AAAAAA;
}

.c26::-moz-placeholder {
  color: #AAAAAA;
}

.c26:-ms-input-placeholder {
  color: #AAAAAA;
}

.c26::-webkit-search-decoration {
  -webkit-appearance: none;
}

.c26::-moz-focus-inner {
  border: none;
  outline: none;
}

.c26:-moz-placeholder,
.c26::-moz-placeholder {
  opacity: 1;
}

.c26::-webkit-input-placeholder {
  font-size: 14px;
}

.c26::-moz-placeholder {
  font-size: 14px;
}

.c26:-ms-input-placeholder {
  font-size: 14px;
}

.c26::placeholder {
  font-size: 14px;
}

.c22 {
  position: relative;
  width: 100%;
}

.c25 {
  position: absolute;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-packjustify: center;
  -webkit-justify: center;
  -ms-flex-packjustify: center;
  justify: center;
  top: 50%;
  -webkit-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  pointer-events: none;
  right: 6px;
}

.c13 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  margin: 6px 0 0;
  padding: 12px 12px 6px;
}

.c13 dt,
.c13 dd {
  margin: 0;
}

.c13 dt:first-of-type,
.c13 dd:first-of-type {
  font-weight: bold;
}

@media only screen and (max-width:768px) {
  .c1 {
    padding: 6px;
  }
}

@media only screen and (max-width:768px) {
  .c2 {
    margin-bottom: 6px;
  }
}

@media only screen and (max-width:768px) {
  .c3 {
    border: solid 1px #EDEDED;
  }
}

@media only screen and (max-width:768px) {
  .c4 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-flex-basis: auto;
    -ms-flex-preferred-size: auto;
    flex-basis: auto;
    -webkit-box-pack: start;
    -webkit-justify-content: flex-start;
    -ms-flex-pack: start;
    justify-content: flex-start;
    -webkit-align-items: stretch;
    -webkit-box-align: stretch;
    -ms-flex-align: stretch;
    align-items: stretch;
  }
}

@media only screen and (max-width:768px) {
  .c4 {
    padding: 6px;
  }
}

@media only screen and (max-width:768px) {

}

@media only screen and (max-width:768px) {

}

@media only screen and (max-width:768px) {

}

@media only screen and (max-width:768px) {

}

@media only screen and (max-width:768px) {
  .c19 {
    padding: 12px;
  }
}

@media only screen and (max-width:768px) {
  .c21 {
    border-bottom: solid 1px rgba(0,0,0,0.33);
  }
}

@media only screen and (max-width:768px) {
  .c27 {
    margin-top: 12px;
  }
}

@media only screen and (max-width:768px) {
  .c30 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c31 {
    border: solid 1px #EDEDED;
  }
}

@media only screen and (max-width:768px) {
  .c31 {
    padding: 24px;
  }
}

@media only screen and (max-width:768px) {
  .c6 {
    padding-right: 6px;
  }
}

@media only screen and (max-width:768px) {
  .c7 {
    border-bottom: solid 1px #EDEDED;
  }
}

@media only screen and (max-width:768px) {
  .c7 {
    padding-bottom: 6px;
  }
}

@media only screen and (max-width:768px) {
  .c24 {
    height: 12px;
  }
}

@media only screen and (max-width:768px) {
  .c32 {
    font-size: 18px;
    line-height: 24px;
    max-width: 432px;
  }
}

@media only screen and (min-width:768px) {
  .c13 dt {
    width: 40%;
  }

  .c13 dd {
    width: 60%;
  }

  .c13 dt:not(:last-of-type),
  .c13 dd:not(:last-of-type) {
    margin-bottom: 6px;
  }

  .c13 dt:first-of-type,
  .c13 dd:first-of-type {
    font-size: 22px;
    line-height: 28px;
    margin-bottom: 12px;
  }
}

@media only screen and (max-width:768px) {
  .c13 dt {
    width: 30%;
  }

  .c13 dd {
    width: 70%;
    text-align: right;
  }

  .c13 dt,
  .c13 dd {
    font-size: 16px;
  }

  .c13 dt:not(:last-of-type),
  .c13 dd:not(:last-of-type) {
    margin-bottom: 3px;
  }

  .c13 dt:first-of-type,
  .c13 dd:first-of-type {
    font-size: 18px;
    line-height: 24px;
    margin-bottom: 6px;
  }
}

<div
  class="c0"
  style="min-height: 100dvh;"
>
  <div
    class="c1"
  >
    <div
      class="c2"
    />
    <div
      class="c3"
    >
      <div
        class="c4"
      >
        <div
          class="c5"
        >
          <div
            class="c6"
          >
            <div
              class="c7"
            >
              <button
                class="c8"
                data-testid="copy-address-icon"
                type="button"
              >
                <svg
                  aria-label="Copy"
                  class="c9"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M9 15h8-8zm0-4h10H9zm0-4h4-4zm7-6v6h6M6 5H2v18h16v-4m4 0H6V1h11l5 5v13z"
                    fill="none"
                    stroke="#000"
                    stroke-width="2"
                  />
                </svg>
              </button>
              <div
                class="c10"
              >
                <span
                  class="c11"
                  style="display: inline-flex; align-items: center; justify-content: space-between; flex: 1;"
                >
                  <span
                    class="c12"
                    style="font-family: "Roboto mono", monospace; letter-spacing: 0;"
                  >
                    <span
                      class="notranslate"
                      translate="no"
                    >
                      oasis1 qz0k 5q8v jqvu 4s4n wxyj 406y lnfl kc4v rcjg huwk
                    </span>
                  </span>
                </span>
              </div>
            </div>
          </div>
          <dl
            class="c13"
            data-testid="account-balance-summary"
          >
            <dt>
              <span
                class="c14"
              >
                Total
              </span>
            </dt>
            <dd
              data-testid="account-balance-total"
            >
              <span>
                <div
                  class="c15"
                  style="display: inline-flex; white-space: nowrap;"
                >
                  100.000001111
                </div>
                <span
                  class="c16"
                >
                  <span
                    class="notranslate"
                    translate="no"
                  >
                     
                  </span>
                </span>
              </span>
            </dd>
            <dt>
              Available
            </dt>
            <dd>
              <span>
                <div
                  class="c15"
                  style="display: inline-flex; white-space: nowrap;"
                >
                  100.0
                </div>
                <span
                  class="c16"
                >
                  <span
                    class="notranslate"
                    translate="no"
                  >
                     
                  </span>
                </span>
              </span>
            </dd>
            <dt>
               
              Staked
            </dt>
            <dd>
              <span>
                <div
                  class="c15"
                  style="display: inline-flex; white-space: nowrap;"
                >
                  0.000001111
                </div>
                <span
                  class="c16"
                >
                  <span
                    class="notranslate"
                    translate="no"
                  >
                     
                  </span>
                </span>
              </span>
            </dd>
            <dt>
              Debonding
            </dt>
            <dd>
              <span>
                <div
                  class="c15"
                  style="display: inline-flex; white-space: nowrap;"
                >
                  0.0
                </div>
                <span
                  class="c16"
                >
                  <span
                    class="notranslate"
                    translate="no"
                  >
                     
                  </span>
                </span>
              </span>
            </dd>
          </dl>
        </div>
      </div>
    </div>
    <div
      class="c2"
    />
    <div
      class="c17"
      style="gap: 24px;"
    >
      <div
        class="c18"
      >
        <div
          class="c3"
        >
          <form>
            <div
              class="c19"
            >
              <div
                class="c2 "
              >
                <label
                  class="c20"
                  for="recipient-id"
                >
                  Recipient
                </label>
                <div
                  class="c21 "
                >
                  <div
                    class="c22"
                  >
                    <input
                      aria-autocomplete="list"
                      aria-expanded="false"
                      autocomplete="off"
                      class="c23"
                      id="recipient-id"
                      name="recipient"
                      placeholder="Enter an address"
                      required=""
                      role="combobox"
                      value=""
                    />
                  </div>
                </div>
              </div>
              <div
                class="c24"
              />
              <div
                class="c2 "
              >
                <label
                  class="c20"
                  for="amount-id"
                >
                  Amount
                </label>
                <div
                  class="c21 "
                >
                  <div
                    class="c22"
                  >
                    <div
                      class="c25"
                    >
                      <span
                        class="c16"
                      />
                    </div>
                    <input
                      autocomplete="off"
                      class="c26"
                      id="amount-id"
                      min="0"
                      name="amount"
                      placeholder="Enter an amount"
                      required=""
                      step="any"
                      type="number"
                      value=""
                    />
                  </div>
                </div>
              </div>
              <div
                class="c24"
              />
              <div
                class="c27"
              >
                <button
                  class="c28"
                  type="submit"
                >
                  Send
                </button>
              </div>
            </div>
          </form>
        </div>
      </div>
      <div
        class="c29"
      >
        <div
          class="c30"
        >
          <div
            class="c31"
          >
            <h3
              class="c32"
            >
              No transactions found.
            </h3>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<AccountPage  /> with missing delegations 1`] = `"Total-Available100.0  Staked-Debonding-"`;
